spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DenseTensor.scala

```diff
@@ -1438,26 +1438,63 @@ private[tensor] class DenseTensor[@specialized T: ClassTag](
       case 0 => s"[${this.getClass.getName} with no dimension]"
       case 1 =>
         val sb = new StringBuilder
-        this.apply1(e => {
-          sb.append(e).append('\n')
-          e
-        })
+        if (this.size().product < 1000) {
```

Comment:
make the 1000 configurable. 

private val foldThreshold = System.getProperty("bigdl.tensor.fold", "1000").toString